lib/awesome_print/formatter.rb

```diff
@@ -318,6 +318,26 @@ def plain_single_line
 
     # Utility methods.
     #------------------------------------------------------------------------------
+    def convert_to_hash(object)
+      if not object.respond_to?(:to_hash)
```

Comment:
I understand this specific instance, but that document is horrible because it gives so little justification for what it specifies.